src/main/webapp/calendar-script.js

```diff
@@ -12,8 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let script = document.createElement('script');
+script.src = 'https://maps.googleapis.com/maps/api/js?key=' + config.API_KEY + 
+  '&libraries=places&callback=initMod';
+script.defer = true;
+script.async = true;
+
+var map;
+var service;
```

Comment:
Add a comment to describe what these are for.

Reply:
It seems like these are just used within the `createMap(...)` function -- can you move them there and change `var` to `const`?